runtime/integration-tests/src/utils/env.rs

```diff
@@ -777,21 +775,39 @@ fn test_env(
 	// Build relay-chain builder
 	let relay = {
 		sp_tracing::enter_span!(sp_tracing::Level::INFO, "Relay - StartUp");
-		let mut state = StateProvider::new(RelayCode.expect("Wasm is build. Qed."));
 
-		// We need to HostConfiguration and use the default here.
+		//TODO(cdamian): Use RelayBlock
+		let mut state =
+			StateProvider::<TFullBackend<centrifuge::Block>, centrifuge::Block>::empty_default(
```

Comment:
@mustermeiszer Shouldn't this fail since we're not using the relay block?

Reply:
mmh. Probably we have the same as the relay.